src/extractor/guidance/intersection_handler.cpp

```diff
@@ -129,12 +131,12 @@ TurnInstruction IntersectionHandler::getInstructionForObvious(const std::size_t
                     const auto distance = util::coordinate_calculation::haversineDistance(
                         coordinates[node_based_graph.GetTarget(via_edge)],
                         coordinates[node_based_graph.GetTarget(road.eid)]);
-                    return {
-                        TurnType::Turn,
-                        (angularDeviation(road.angle, STRAIGHT_ANGLE) < FUZZY_ANGLE_DIFFERENCE ||
-                         distance > 2 * MAX_COLLAPSE_DISTANCE)
-                            ? DirectionModifier::Straight
-                            : getTurnDirection(road.angle)};
+
+                    return {TurnType::Turn,
+                            (angularDeviation(road.angle, STRAIGHT_ANGLE) < NARROW_TURN_ANGLE &&
```

Comment:
we return a straight below 40 degree instead of only below 25 degree now? isn't the issue exactly the other way around: we return a straight but the user wants a turn? (from the original wording in the issue it's a bit unclear to me what the user expects to be honest)